tests/tas-all-amps/pom.xml

```diff
@@ -195,14 +195,15 @@
                                             <overWrite>false</overWrite>
                                             <outputDirectory>${project.build.directory}/amps_share</outputDirectory>
                                         </artifactItem>
+                                        <!-- Incompatible with MM 1.4.1-RC1. Jira issue -> https://issues.alfresco.com/jira/browse/REPO-5133
```

Comment:
As we know, we should prioritise REPO-5133 asap to update S3/Glacier version (for ACS 6.2.1+) so that these does not remain commented-out for too long.

Reply:
Yes, I agree.